css/index.css

```diff
@@ -1 +1,405 @@
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+/* General */
+html {
+    font-size: 62.5%;
+}
+
+.container {
+    width: 95%;
+    max-width: 880px;
+    margin: 0 auto;
+    font-size: 1.6rem;
+}
+
+header {
+	width: 100%;
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	align-items: center;
+	margin: 28px 0;
+}
+
+nav {
+	width: 69%;
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+}
+
+a {
+	text-decoration: none;
+	color: gray;
+	font-size: 1.5rem;
+}
+
+h1 {
+	font-size: 7.2rem;
+}
+
+h2 {
+	line-height: 1.1;
+}
+
+p {
+	line-height: 1.1;
+}
+
+footer {
+	margin: 36px 0;
+	text-align: center;
+	font-size: 1.6rem;
+}
+
+/* Home Page CSS */
+
+.home {
+	display: flex;
+	flex-direction: row;
+	margin-top: 30px;
+}
+
+.top {
+    text-align: center;
+    justify-content: space-around;
+    margin-top: 70px;
+}
+
+.home-inner {
+	display: flex;
+	flex-direction: column;
+    justify-content: center;
+    flex-grow: 1;
+    align-items: center;
+    padding-left: 6%;
+}
+
+.middle-img {
+	width: 100%;
+	margin-top: 30px;
+}
+
+.middle {
+	border-top: 1px solid #000;
+    padding-top: 40px;
+    margin-top: 75px;
+}
+
+.bottom {
+	padding-bottom: 44px;
+    border-bottom: 1px solid black;
+    justify-content: space-between;
+}
+
+.bottom section {
+    width: 33%;
+}
+
+.home-btn {
+	width: 37%;
+	border: 1px solid #000;
+    padding: 7px 0;
+    margin-top: 8px;
+}
+
+address {
+    margin-top: 32px;
+}
+
+/* Services CSS */
+
+h3 {
+	font-size: 2.8rem;
+}
+
+.services-btn span {
+	font-size: 1.2rem;
+}
+
+ul {
+	list-style-position: 
+}
+
+li::before {
+	content: "•"; color: black;
+	line-height: 2;
+}
+
+li span {
+	position: relative;
+	left: 10px;
+}
+
+.services {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	margin: 40px 0;
+}
+
+.services-btn {
+	width: 35%;
+	border: 1px solid #000;
+	padding: 10px 0;
+	margin-top: 20px 0;
+	border-radius: 5px;
+}
+
+.second {
+	display: flex;
+	flex-direction: column;
+	border-top: 1px solid #000;
+	border-bottom: 1px solid #000;
+	padding: 20px 0;
+}
+
+.row {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+}
+
+.row section {
+	background-color: #F5F5F5;
+	margin: 18px 0;
+	width: 48%;
+	border: 1px solid black;
+	padding: 20px 2%;
+}
+
+.services-inner {
+	width: 45%;
+}
+
+
+/* Tablet */
+
+@media only screen and (max-width: 800px) {
+
+    .container {
+		  width: 90%;
+		  max-width: 656px;
+      }
```

Comment:
There are some indentation issues; see: [prettier](https://prettier.io/)! It's really good.